db/models/user.js

```diff
@@ -2,27 +2,48 @@
 
 // bcrypt docs: https://www.npmjs.com/package/bcrypt
 const bcrypt = require('bcryptjs')
-    , {STRING, VIRTUAL} = require('sequelize')
+    , {BOOLEAN, STRING, VIRTUAL} = require('sequelize')
 
 module.exports = db => db.define('users', {
-  name: STRING,
+  userName: STRING,
```

Comment:
Think we could just have users log in with an email and a password -- username might be overkill, especially since we are storing firstName and lastName also. Thoughts?